### PR TITLE
Fix/535/do not create empty revision

### LINKED ISSUE #535: Create a new version of the file during an empty modification
When you switch to the edit mode of the file and validate a modification without having made any changes in the file, a new version of the file will still create it based on the previous version. This may eventually overload the page with "useless" version.

### PR DESCRIPTION
**Pull request by inkhey**
Do not create new empty revision when content do not change.
Fix #535

### CODE REVIEW

File: tracim/tracim/lib/webdav/__init__.py [hunk 1]
```diff
@@ -5,11 +5,14 @@
...
 
 from tracim.lib.content import ContentApi
+from tracim.lib.utils import SameValueError
+from tracim.lib.base import logger
 from tracim.model import new_revision
 from tracim.model.data import ActionDescription
 from tracim.model.data import ContentType
 from tracim.model.data import Content
 from tracim.model.data import Workspace
+from wsgidav.dav_error import DAVError, HTTP_FORBIDDEN, HTTP_NOT_MODIFIED
```
Comment (buxx) [inline tracim/tracim/lib/webdav/__init__.py]: `import` code style to follow: "Imports" chapter in http://algoo.trac.im/workspaces/11/folders/1194/pages/2043

**Comment by buxx** [comment]
Does exist a unit test who check the case (do not create an empty revision) ?

File: tracim/tracim/lib/webdav/__init__.py (hunk 1 above)
Comment (inkhey) [reply tracim/tracim/lib/webdav/__init__.py]: HTTP_NOT_MODIFIED is not used anymore, i drop it.

**Comment by inkhey** [comment]
@buxx Now, there is !

**Pull request merged**